sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -38,14 +46,29 @@ public enum ServiceVersion
             V2019_02_02 = 1,
         }
     }
-    public partial class TableEntity
+    public partial class TableEntity : Azure.Data.Tables.ITableEntity
     {
         public TableEntity() { }
+        public TableEntity(string partitionKey, string rowKey) { }
         public string ETag { get { throw null; } set { } }
         public string PartitionKey { get { throw null; } set { } }
         public string RowKey { get { throw null; } set { } }
         public System.DateTimeOffset Timestamp { get { throw null; } set { } }
     }
+    public partial class TableQuery<TElement> : System.Collections.Generic.IEnumerable<TElement>, System.Collections.IEnumerable, System.Linq.IQueryable, System.Linq.IQueryable<TElement>
+    {
+        public TableQuery() { }
+        public System.Type ElementType { get { throw null; } }
+        public System.Linq.Expressions.Expression Expression { get { throw null; } }
+        public string FilterString { get { throw null; } set { } }
```

Comment:
Do we need to have these members public and settable. I expected they are going to be initialized by using Linq operators like Select or Where or skip etc.

Reply:
Agreed - this could also be set the same as one would pass arguments to `Query`, but that seems unnecessary and potentially confusing